Set 5th parameter to 0 to change the dropdown menu behavior
to that of Fl_Menu_Button().

--- a/fltk/src/Fl_Choice.cxx
+++ b/fltk/src/Fl_Choice.cxx
@@ -186,13 +186,13 @@ int Fl_Choice::handle(int e) {
   J1:
     if (Fl::scheme()
 	|| fl_contrast(textcolor(), FL_BACKGROUND2_COLOR) != textcolor()) {
-      v = menu()->pulldown(x(), y(), w(), h(), mvalue(), this);
+      v = menu()->pulldown(x(), y(), w(), h(), 0, this);
     } else {
       // In order to preserve the old look-n-feel of "white" menus,
       // temporarily override the color() of this widget...
       Fl_Color c = color();
       color(FL_BACKGROUND2_COLOR);
-      v = menu()->pulldown(x(), y(), w(), h(), mvalue(), this);
+      v = menu()->pulldown(x(), y(), w(), h(), 0, this);
       color(c);
     }
     if (!v || v->submenu()) return 1;
